{{cookiecutter.python_name}}/setup.py

```diff
@@ -62,6 +58,9 @@
 
 long_description = (HERE / "README.md").read_text()
 
+# Get the package info from package.json
+pkg_json = json.loads((HERE / "package.json").read_bytes())
```

Comment:
I thought it might be clearer to put `pkg_json` closer to its usage. Happy to revert if it doesn't suit you.

Reply:
- # - you```
```
- ## - ```_**@[]()````**_```
```
- ```**I_```

```_**

Reply:
- # ````
```

```
````